circleguard/enums.py

```diff
@@ -267,6 +272,52 @@ class Mod():
              V2, TD, # we stop caring about order after this point
              FI, RD, CN ,TP, K1, K2, K3, K4, K5, K6, K7, K8, K9, CO, MR]
 
+    # same as ORDER but with NM included
+    ALL = [NM, EZ, HD, HT, DT, _NC, HR, FL, NF, SD, _PF, RX, AP, SO, AT,
+           V2, TD, FI, RD, CN ,TP, K1, K2, K3, K4, K5, K6, K7, K8, K9, CO, MR]
```

Comment:
These two array can be combined.
No reason to not have NM in order as far as I can see

Reply:
hm, I was thinking that `NM` wouldn't make much sense in `ORDER` because nobody writes +NM in scorepost titles, but it wouldn't break anything to change it and makes sense coding wise